ddtrace/llmobs/_llmobs.py

```diff
@@ -88,7 +89,13 @@ def __init__(self, tracer=None):
             interval=float(os.getenv("_DD_LLMOBS_WRITER_INTERVAL", 1.0)),
             timeout=float(os.getenv("_DD_LLMOBS_WRITER_TIMEOUT", 5.0)),
         )
-        self._trace_processor = LLMObsTraceProcessor(self._llmobs_span_writer)
+
+        self._evaluator_runner = EvaluatorRunner(
```

Comment:
Are we going to make this feature enabled by default? If a user doesn't care about ragas evaluations, does it make sense for us to keep a background thread constantly running?

Reply:
Your right I don't think it makes sense. I was following the pattern with eval metrics where the eval metric writer is started even though a user may not submit eval metrics but that's a different case. I'll add a check inside the runner to see there are any evaluators turned on and if not, we don't start the runner